solutions/src/test/java/ru/sberbank/school/task10/ConcurrentScalableThreadPoolTest.java

```diff
@@ -0,0 +1,166 @@
+package ru.sberbank.school.task10;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.concurrent.*;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ConcurrentScalableThreadPoolTest {
+    private int max = 10;
+    private int core = 3;
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new ConcurrentScalableThreadPool(core, max);
+    }
+
+    private Runnable newRunnable(AtomicInteger counter, CountDownLatch latch) {
+        return () -> {
+
+                System.out.println("Task: " + counter.incrementAndGet()
+                        + " - Thread: " + Thread.currentThread().getName());
+
+            latch.countDown();
+        };
+    }
+
+    private Callable<String> newCallable(AtomicInteger counter, CountDownLatch latch) {
+        return () -> {
+            String res;
+                res = "Task: " + counter.incrementAndGet() + " - Thread: " + Thread.currentThread().getName();
```

Comment:
&#39;block&#39; child has incorrect indentation level 16, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a218691332>*